ui/job-view/App.jsx

```diff
@@ -228,7 +228,9 @@ class App extends React.Component {
   };
 
   setCurrentRepoTreeStatus = (status) => {
-    const link = document.head.querySelector('link[rel="shortcut icon"]');
+    const link = document.head.querySelector(
```

Comment:
Should you be adding a new block rather than changing this? If I'm reading correctly, getting rid of the update to the existing element will stop updating something on Push Health.

Reply:
The Push Health favicon gets set at https://github.com/mozilla/treeherder/blob/4a656b41af00d98e297b1488694c41e714c48a62/ui/push-health/MyPushes.jsx#L177-L180 and https://github.com/mozilla/treeherder/blob/4a656b41af00d98e297b1488694c41e714c48a62/ui/push-health/MyPushes.jsx#L177-L180 It doesn't get set from JavaScript as far as I can see.

Reply:
Yes, it was broken since the conversion to React.